README.md

```diff
@@ -236,3 +234,47 @@ $ chembl_downloader --test
+|              7   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              8   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|              9   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             10   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             11   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             12   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             13   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             14   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             15   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             16   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             17   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             18   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             19   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             20   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             21   | Yes                  |                                                                                                 |
+|             22   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             22.1 | No                   | <urlopen error ftp error: error_perm('550 Failed to change directory.')>                        |
+|             23   | Yes                  |                                                                                                 |
+|             24   | No                   | <urlopen error ftp error: URLError("ftp error: error_perm('550 Failed to change directory.')")> |
+|             24.1 | No                   | <urlopen error ftp error: error_perm('550 Failed to change directory.')>                        |
```

Comment:
should be 24_1 when formatting in FTP address

Reply:
Yes. It is adjusted that way when passed to ChEMBL downloader. I can be explicit here as well if needed.

Reply:
I took care of this upstream